psq/queue.py

```diff
@@ -64,7 +69,11 @@ def _get_or_create_subscription(self):
         if not subscription.exists():
```

Comment:
That's fair, but I'd rather pay the one-time cost of a check (this only happens at startup) than depend on the API catching the conflict. It's a bit more defensive.